src/main/main-window.ts

```diff
@@ -85,7 +89,7 @@ export function createMainWindow(
     //icon: getAppIcon(),
     webPreferences: {
       nodeIntegration: true,
-      enableRemoteModule: true
+      contextIsolation: false
```

Comment:
This is may be fine since it's only for the main window, but should we try to retain context isolation? Should be doable with some modifications to the preload scripts and ipc calls.

Reply:
Yeah this is a good idea. I'll put it on the board.